### ISSUE TITLE
Blue focus outline when editing title in sidebar is missing

### ISSUE THREAD

**Issue by tommoor** [outline/outline#10087]
It's getting cropped now, this was working correctly previously.

**Comment by ShahilMd** [comment]
@tommoor, Hey, can you explain  little bit more in which section of the sidebar Blue focus outline, or can you please add an image reference
